scss/_variables.scss

```diff
@@ -534,6 +534,7 @@ $h3-font-size:                $font-size-base * 1.5 !default;     // 24px
 $h4-font-size:                $font-size-xlg !default;            // 20px
 $h5-font-size:                $font-size-lg !default;             // 18px
 $h6-font-size:                $font-size-base !default;           // 16px
+$h7-font-size:                $font-size-base * .825 !default;    // 14px
```

Comment:
In the HTML spec, "The h1 element is said to have the highest rank, the h6 element has the lowest rank" so it is weird to have a `h7` somewhere. Moreover, you probably don't need this new variable and could use directly `$font-size-sm` to do:

```scss
$font-sizes: (
  // [...]
  7: $font-size-sm,
) !default;
```

In `/docs/5.0/utilities/text/#font-size`, `.fs-7 text` is not mentionned.
And in the paragraph, it would be important to add a note explaining that there is a `fs-7` event if `h7` doesn't exist. Something like that.

Reply:
Yes, totally my bad for h7, I wasn't happy to do so either, so yes I'll directly use font-size-sm